.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,7 +1,11 @@
 ---
 name: Bug report
 about: Create a report to help us improve
+<<<<<<< HEAD
 title: "[Bug] Give a summary of the problem"
+=======
+title: "[Bug] Summary of problem"
+>>>>>>> Update issue templates
```

Comment:
A merge conflict hasn't been solved here.

The "=====" line is the divider, the lines up until the "<<<<" line is what existed (ie HEAD), the lines down until ">>>" is new in "Update issue templates"  branch(?)

To you the task of editing the file, deleting all bad lines and the administrative lines (===, <<<, and >>>), and editing the changed lines to what it should be.

Normally you do this before adding a file with a conflict to git staging